polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -1425,6 +1452,57 @@ pub mod pallet {
 				weight_limit,
 			)
 		}
+
+		/// Authorize another `aliaser` location to alias into the local `origin` making this call.
+		///
+		/// Usually useful to allow your local account to be aliased into from a remote location
+		/// also under your control (like your account on another chain).
+		///
+		/// WARNING: make sure the caller `origin` (you) trusts the `aliaser` location to act in
+		/// their/your name. Once authorized using this call, the `aliaser` can freely impersonate
+		/// `origin` in XCM programs executed on the local chain.
+		#[pallet::call_index(14)]
+		pub fn add_authorized_alias(
+			origin: OriginFor<T>,
+			aliaser: Box<VersionedLocation>,
```

Comment:
maybe we could allow also multiple items at once e.g.  `aliasers: Box<Vec<VersionedLocation>>,` but in the real scenarios probably it will be called one-by-one, so no need to change here